SYCL/Basic/buffer/buffer_dev_to_dev.cpp

```diff
@@ -25,9 +25,13 @@ int main() {
                           {property::buffer::use_host_ptr()});
 
     default_selector Selector;
+    device Device(Selector);
 
-    queue FirstQueue(Selector);
-    queue SecondQueue(Selector);
+    context FirstContext(Device);
+    context SecondContext(Device);
```

Comment:
Minor NIT: is it possible to use default ctx instead of one of explicit ones?

Reply:
Yes, if we have two distinct devices (which is not always the case). The reason test requires two different contexts is that it forces us to move data from device to device through host.